addon/styles/_frost-detail-subtabs.scss

```diff
@@ -6,6 +6,9 @@ $frost-detail-tab-inactive: $frost-color-grey-5;
   display: flex;
   flex-direction: column;
   align-items: center;
-  width: 100px;
-  margin: 20px 0 0 20px;
+  width: 120px;
+  margin: 10px 0 ;
+  padding: 0 10px 0 20px;
```

Comment:
This doesn't appear equivalent to the previous spacing - the right padding and removal of 10px off the top margin